lib/gs_schemas/capnp/game_types.capnp

```diff
@@ -102,3 +102,21 @@ struct FullChunkData {
     blockPalette @0 :List(UInt64);
     blockData @1 :List(UInt16);
 }
+
+# Action Data
+struct PositionData {
```

Comment:
The raycast should happen on the client and only the block position should be sent in the network request, syncing position+orientation can happen separately and be used just for validation